mern-ecommerce/server.js

```diff
@@ -25,7 +25,7 @@ const connectDB = async () => {
       useNewUrlParser: true,
       useUnifiedTopology: true,
       useCreateIndex: true,
-      useFindAndModify: false,
+      useFindAndModiify: false,
```

Comment:
The issue here is a typo in the option name. It should be `useFindAndModify` instead of `useFindAndModiify`. The correct line should be `useFindAndModify: false`.

Reply:
@CodeHawk what do you mean? Care to explain further on this?

Reply:
You have a typo in the option name. It should be `useFindAndModify` instead of `useFindAndModiify`.